ReactiveCocoa/Swift/SignalProducerExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SignalProducerExtensions.swift
+//  ReactiveCocoa
+//
+//  Created by Nate Stedman on 3/22/16.
+//  Copyright © 2016 GitHub. All rights reserved.
+//
+
+import enum Result.NoError
+
+extension SignalProducer {
+	func demoteErrors() -> SignalProducer<Value, NoError> {
+		return flatMapError { _ in SignalProducer<Value, NoError>.empty }
```

Comment:
Or perhaps rename the function to `ignoreErrors`, for consistency with `ignoreNil`